### DISCUSSION TITLE
How to Issue Certificates for Multiple Domains

### DISCUSSION THREAD

**Discussion by XIYO** [acmesh-official/acme.sh#5175] [Q&A] [answered]
Dear Community,

I hope this message finds you well. I've been exploring the capabilities of ACME with the help of GPT, but I haven't found a clear answer yet, so I'm turning to you for assistance.

I am currently managing two web services on my server, which are associated with two domains: a.com and b.com. Both domains are registered with Cloudflare. Presently, I manually update using tokens, account_id, and zone_id.

My goal is to automate this process. I'm interested in using the --install-cron option with ACME; however, each domain uses different tokens and IDs. Is there a feature that allows registering a crontab for domains that use different tokens?

One solution I'm considering is to write a script that references each environment variable individually. Nevertheless, I would appreciate any suggestions for a better method or more insights into ACME's features.

Thank you for reading, and I wish you a wonderful day!

**Comment by neilpang** [comment, marked as answer]
see https://github.com/acmesh-official/acme.sh/wiki/dnsapi#1-cloudflare-option
